InternalPythonModules/GPX_Module/GPX_Parser_Module.py

```diff
@@ -75,158 +80,178 @@ def getModuleDescription(self):
     def getModuleVersionNumber(self):
         return "1.2"
 
-    def isDataSourceIngestModuleFactory(self):
+    def isFileIngestModuleFactory(self):
         return True
 
-    def createDataSourceIngestModule(self, ingestOptions):
+    def createFileIngestModule(self, ingestOptions):
         return GPXParserDataSourceIngestModule()
 
-    
+
 # Data Source-level ingest module. One gets created per data source.
-class GPXParserDataSourceIngestModule(DataSourceIngestModule):
+class GPXParserDataSourceIngestModule(FileIngestModule):
```

Comment:
We should rename all class names - GPXParserDataSourceIngestModuleFactory, GPXParserDataSourceIngestModule - to reflect that it's a file ingest module, not data source ingest module. Also, there are comments that also talk about data source ingest.

Reply:
Got it.  That should be integrated now.